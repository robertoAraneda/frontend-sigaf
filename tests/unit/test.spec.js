import Button from '@/components/base/Btn.vue'
import { mount } from '@vue/test-utils'

test('Montar el componente Base Button', () => {
  const wrapper = mount(Button)

  expect(wrapper.html()).toMatchSnapshot()
})
