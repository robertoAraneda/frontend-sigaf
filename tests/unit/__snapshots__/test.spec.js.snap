// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Montar el componente Base Button 1`] = `
<button type="button" class="mr-2 v-btn v-btn--flat v-btn--text theme--light v-size--default blueS--text"><span class="v-btn__content"><i aria-hidden="true" class="v-icon notranslate v-icon--left mdi mdi-alert-circle theme--light" style="font-size: 30px;"></i>
  Error
</span></button>
`;
